src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -224,22 +275,28 @@ const FileViewer = ({ file, onClose, onDownload, downloader, show, progress }: F
               className="outline-none z-10 flex max-h-full max-w-full flex-col items-start justify-start overflow-auto"
             >
               <div onClick={(e) => e.stopPropagation()} className="">
+                <button
+                  className="absolute top-1/2 left-10 rounded-full bg-black p-4 text-white"
+                  onClick={() => changeFile('prev')}
+                >
+                  <CaretLeft size={24} />
+                </button>
                 {blob ? (
                   <Suspense fallback={<div></div>}>
-                    <Viewer blob={blob} />
+                    <Viewer blob={blob} changeFile={changeFile} />
                   </Suspense>
-                ) : (
+                ) : progress ? (
```

Comment:
With this, when the progress is 0 the Download button will be displayed instead of the progress bar, if the component has to be displayed when the progress is less than 1 it would be better to change to progress < 1. To avoid flashing of the download button